spec/requests/case_spec.rb

```diff
@@ -2,6 +2,30 @@
 require 'support/shared_examples_for_glimr'
 
 RSpec.describe GlimrApiClient::Case do
+  include_examples 'no fees', 'TT/2016/00001', 'ABC123'
+
+  subject { described_class.find('TT/2016/00001', 'ABC123') }
+
+  describe '#title' do
+    it 'returns the title' do
```

Comment:
Could I suggest "defaults to 'Missing Title' if GLiMR doesn't return a title" as 'Missing Title' isn't actually the title of the case.